build.gradle

```diff
@@ -33,4 +33,10 @@ task runBankocrAdriana (dependsOn: 'classes', type: JavaExec) {
     main = 'jalau.at18.katas.bankocr.adriana.Main'
     standardInput=System.in
     classpath = sourceSets.main.runtimeClasspath
+}
+    
+task runBankOcr (dependsOn: 'classes', type: JavaExec) {
```

Comment:
This is optional, you don't need to add task to run it for now as we are mainly doing the testing with unit tests